### PR TITLE
Rearrange Contributing pages + add gitpod info to contributing area

### PR DESCRIPTION
**Pull request by reshamas**
Towards #178

- move Event Info (PR section) into "Contributing" section
- add Gitpod info to "Contributing" section

:books: Documentation preview en :books:: https://pymc-data-umbrella--216.org.readthedocs.build/en/216/

:books: Documentation preview es :books:: https://pymc-data-umbrella-es--216.org.readthedocs.build/es/216/

:books: Documentation preview pt :books:: https://pymc-data-umbrella-pt--216.org.readthedocs.build/pt/216/

### CODE REVIEW

File: contributing/tutorials/docstring_tutorial.md
Comment (OriolAbril) [inline contributing/tutorials/docstring_tutorial.md:1]: Is this being duplicated or moved? Reviewing from my phone and not sure what is going on

Comment (reshamas) [reply contributing/tutorials/docstring_tutorial.md:1]: @OriolAbril 
I am trying to move the "environment setup" section from "Event info" to "Contribute".

Comment (reshamas) [reply contributing/tutorials/docstring_tutorial.md:1]: @OriolAbril 
Perhaps I should not have "moved" them and instead changed how I "referenced" them?

**Comment by reshamas** [comment]
@OriolAbril 
Hopefully I did not make too much of a mess. The structure is what I am looking for.

I hope moving files around does not interfere with the translations.  Not sure what the impact is.

File: contributing/tutorials/docstring_tutorial.md
Comment (OriolAbril) [reply contributing/tutorials/docstring_tutorial.md:1]: Moving is fine, they look moved now in yhe PR changes. The only step missing is adding the redirects so links we have shared to their current location continue working after we merge this PR.

As we use `ref` type cross references, all internal links should continue to work without any extra work needed on the files that reference the updated files.

File: conf.py [hunk 1]
```diff
@@ -59,7 +59,7 @@
     "webinars/contributing_to_pymc/index.md": "about/contributing_to_pymc/index.md",
     "webinars/intro_to_array_operations/index.md": "about/intro_to_array_operations/index.md",
     "webinars/probabilistic_programming_with_pymc/index.md": "about/probabilistic_programming_with_pymc/index.md",
-    "sprint/docstring_tutorial.md": "sprint/tutorials/docstring_tutorial.md",
+    "sprint/docstring_tutorial.md": "contributing/tutorials/docstring_tutorial.md",
```
Comment (OriolAbril) [inline conf.py:62]: This is great

File: conf.py [hunk 2]
```diff
@@ -59,7 +59,7 @@
...
     "index.md": "2023-03_sprint/index.md",
```
Comment (OriolAbril) [inline conf.py:63]:
```suggestion
    "index.md": "2023-03_sprint/index.md",
    "sprint/tutorials/docstring_tutorial.md": "contributing/tutorials/docstring_tutorial.md",
```

And this needs to be added, also for env setup and pr step by step

**Comment by OriolAbril** [comment]
I am not sure about effect to translations. If it doesn't catch the rename and loses the info I'll rename the po files to keep that info

**Pull request merged**